build.gradle

```diff
@@ -290,23 +286,12 @@ task unpack(type: Copy) {
 }
 
 def generatedOutputDir = 'src/generated/java'
-
-// TODO: we can remove lines 297 & 298 after everyone has run the latest version of the code on their local environments
 clean.doFirst {
     delete("${projectDir}/$generatedOutputDir")
     delete("${rootDir}/datarepo-client/build")
     delete("${rootDir}/datarepo-client/generated")
 }
 
-def excludePattern = 'generated-src/antlr/main/bio/terra/grammar/'
-def excludeGrammarPattern = '${buildDir}/' + excludePattern + '*'
-
-
-tasks.withType(Checkstyle) {
-    exclude excludeGrammarPattern
-    exclude "${buildDir}/generated-src/antlr/main/bio/terra/grammar/*"
-}
-
```

Comment:
Does this mess with the antlr functionality?

Reply:
nope! this is accounted for already in `checkstyle-suppressions.xml`